controllers/metal3.io/preprovisioningimage_controller.go

```diff
@@ -214,6 +293,20 @@ func setImage(generation int64, status *metal3.PreprovisioningImageStatus, url s
 	return changed
 }
 
+func setUnready(generation int64, status *metal3.PreprovisioningImageStatus, message string) bool {
+	newStatus := status.DeepCopy()
+
+	time := metav1.Now()
+	reason := reasonImageSuccess
+	setImageCondition(generation, newStatus,
+		metal3.ConditionImageReady, metav1.ConditionFalse,
+		time, reason, message)
+
+	changed := !apiequality.Semantic.DeepEqual(status, &newStatus)
```

Comment:
Just wondering why this is necessary. The return value isn't currently being used.

Reply:
Just for consistency with the others. Originally I was using the return value, but since we only call this if it's going to return true I figured it would be clearer not to.

Reply:
Makes sense.